command/crypto/otp/generate.go

```diff
@@ -17,10 +17,12 @@ import (
 
 func generateCommand() cli.Command {
 	return cli.Command{
-		Name:        "generate",
-		Action:      command.ActionFunc(generateAction),
-		Usage:       "generate a one-time password",
-		UsageText:   `**step crypto otp generate**`,
+		Name:   "generate",
+		Action: command.ActionFunc(generateAction),
+		Usage:  "generate a one-time password",
+		UsageText: `**step crypto otp generate** [**--issuer**=<name>]
+[**account**=<user-name>] [**--period**=<seconds>] [**--length**=<size>]
+[**--alg**=<alg>] [*--time**=<time|duration>] [**--url**] [**--qr**]`,
```

Comment:
Is this gofmt?

Reply:
And `[**account**=<user-name>]` has the `--` missing.

Reply:
I've just run `gofmt` on it. Don't think there were any changes. Also, `make lint` was passing.

Reply:
I'm just saying `gofmt` because before the values were aligned, and now they are not.

Reply:
Ok it looks like when a string with backticks takes more than one line, then it breaks the alignment. So it's ok.